[2025-01-02T20:10:21.995+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-02T20:10:22.017+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: instacart_etl.etl_dim_order_products manual__2025-01-02T20:03:41.987678+00:00 [queued]>
[2025-01-02T20:10:22.027+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: instacart_etl.etl_dim_order_products manual__2025-01-02T20:03:41.987678+00:00 [queued]>
[2025-01-02T20:10:22.028+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-02T20:10:22.062+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): etl_dim_order_products> on 2025-01-02 20:03:41.987678+00:00
[2025-01-02T20:10:22.073+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=762) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-02T20:10:22.074+0000] {standard_task_runner.py:72} INFO - Started process 764 to run task
[2025-01-02T20:10:22.073+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'instacart_etl', 'etl_dim_order_products', 'manual__2025-01-02T20:03:41.987678+00:00', '--job-id', '81', '--raw', '--subdir', 'DAGS_FOLDER/instacart_etl_dag.py', '--cfg-path', '/tmp/tmps_tkv9r8']
[2025-01-02T20:10:22.075+0000] {standard_task_runner.py:105} INFO - Job 81: Subtask etl_dim_order_products
[2025-01-02T20:10:22.130+0000] {task_command.py:467} INFO - Running <TaskInstance: instacart_etl.etl_dim_order_products manual__2025-01-02T20:03:41.987678+00:00 [running]> on host 70b53016fb73
[2025-01-02T20:10:22.225+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='instacart_etl' AIRFLOW_CTX_TASK_ID='etl_dim_order_products' AIRFLOW_CTX_EXECUTION_DATE='2025-01-02T20:03:41.987678+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-02T20:03:41.987678+00:00'
[2025-01-02T20:10:22.227+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-02T20:16:11.779+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-02T20:16:11.873+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-02T20:16:11.876+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=instacart_etl, task_id=etl_dim_order_products, run_id=manual__2025-01-02T20:03:41.987678+00:00, execution_date=20250102T200341, start_date=20250102T201022, end_date=20250102T201611
[2025-01-02T20:16:11.951+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-02T20:16:12.024+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-02T20:16:12.030+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
